### ISSUE TITLE
[Feature request] FUTO Polycentric support

### ISSUE THREAD

**Issue by TheTrueWhiteOwl** [iv-org/invidious#5028]
**Is your feature request related to a problem? Please describe.**

On Materialious there is a feature request for supporting the polycentric comments available on GrayJay Materialious/Materialious#424. I believe this is a great idea allowing the Google-free alternatives to be more connected. (And it could allow posting comments on Invidious without using YouTube).

The reason I am passing this feature request from Materialious on to Invidious is because this feature could benefit everyone who uses the Invidious API rather than just the Materialious client.

**Describe the solution you'd like**

Similarly to the Reddit comments allow an additional source of comments. This could be added to the invidious api without any major issues.
How to fetch these comments? No clue, that is what must be discussed in this issue.

Posting comments could probably also be implemented by using polycentric, but that would definitely require more work and I think it would be more interesting to first expand the options when it comes to comment sources.

Polycentric is also an open-source project available at [futo-org/polycentric](https://github.com/futo-org/polycentric).

**Comment by syeopite** [comment]
With Reddit's crackdown on datacenter IPs it is pretty much impossible to access Reddit comments on most public instances.

As Unixfox said this feature is only present because it was there before the current dev team took over the project.

But the feature is honestly out of scope for invidious as a pure alternative frontend to YouTube.

It is something I've been thinking of deprecating — or at least making it an opt-in feature through something like a compile time flag. It is barely maintained anyways.

**Comment by mk-pmb** [comment]
Yes for deprecation. Emote 🚀 on this post to vote for jettisoning the reddit comment integration.
